Model/Config.php

```diff
@@ -83,6 +83,12 @@ public function getHost() {
         case 'ug':
             return $this->getApiHost().'.ug';
             break;
+        case 'dz':
+            return $this->getApiHost().'.dz';
+            break;
+        case 'sn':
+            return $this->getApiHost().'.sn';
+            break;
         default:
             return "";
```

Comment:
@bpaz unrelated to this change, but I think this should be single quotes `return '';` to follow the same convention